### ISSUE TITLE
Did it get used?

### ISSUE THREAD

**Issue by yarikoptic** [NCCR-SYNAPSY/neurodatapub#39]
Hi @sebastientourbier , I am curious ;-)

**Comment by sebastientourbier** [comment]
Hi @yarikoptic ! Long time :)

To answer straight to your question: I do not think so...

Here is the context of development for this tool:

For an old 10years research project (NCCR-SYNAPSY), I designed a data management plan that I was teaching and coordinating researchers to adopt which mainly relied on:
- BIDS for data organization and
- Datalad as Data versioning system.

I developed this tool in this context as we wanted to reference a collection of datasets produced by this research project on GitHub, still keeping our data safe in machine accessible via SSH.

Adopting Datalad might be scary for non-IT experts and the goal of this tool was to lower the barriers for non-IT data managers in the project to make the publication process via a GUI, and I was not aware of datalad-gooey at this time.

However, after about one year in the project, I had another job opportunity and I am afraid this effort got lost.
